src/components/Login/LoginContainer.tsx

```diff
@@ -144,17 +145,26 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     }
   }, [isLoading, endpoint, navigation, token, features]);
 
+  const setState = useState()[1];
   useEffect(() => {
-    const skipScanningIfParamsInDeepLink = async (): Promise<void> => {
+    const skipScanningIfParamsInDeepLink = async (): Promise<any> => {
       const { queryParams } = await Linking.parseInitialURLAsync();
+      const queryEndpoint = queryParams?.endpoint;
+      if (queryEndpoint && !RegExp(DOMAIN_CHECK).test(queryEndpoint)) {
+        const error = new Error(`Invalid endpoint: ${queryEndpoint}`);
+        Sentry.captureException(error);
```

Comment:
Don't need this since it'll be called in the error boundary.

Reply:
ok~